src/app/__init__.py

```diff
@@ -1,26 +1,27 @@
-import os
 
-from dotenv import load_dotenv
+from app import routes
 from flask import Flask, request
+from dotenv import load_dotenv
+
+from .models.timelinepost import TimelinePost
+import os
+
+
 from peewee import *
 
 load_dotenv()
 app = Flask(__name__)
-app.config["SECRET_KEY"] = os.getenv("APP_SECRET")
+# app.config["SECRET_KEY"] = os.getenv("APP_SECRET")
 
 mydb = MySQLDatabase(
     os.getenv("MYSQL_DATABASE"),
     user=os.getenv("MYSQL_USER"),
     password=os.getenv("MYSQL_PASSWORD"),
     host=os.getenv("MYSQL_HOST"),
-    port=3306,
+    port=4000,
```

Comment:
Why a change in the port is required?

Reply:
Sorry, my local machine uses port 4000, I will change this to 3306 and commit